spec/classes/foreman_proxy__plugin__host_reports_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'spec_helper'
+
+describe 'foreman_proxy::plugin::host_reports' do
+  on_plugin_os.each do |os, os_facts|
+    context "on #{os}" do
+      let(:facts) { os_facts }
+      let(:pre_condition) { 'include foreman_proxy' }
+
+      describe 'with default settings' do
+        it { should contain_foreman_proxy__plugin__module('host_reports') }
+        it 'host_reports.yml should contain the correct configuration' do
+          verify_exact_contents(catalogue, '/etc/foreman-proxy/settings.d/host_reports.yml', [
+            '---',
+            ':enabled: https',
+            ':reported_proxy_hostname: foo.example.com',
+            ':debug_payload: false',
+            ':spool_dir: /var/lib/foreman-proxy/host_reports',
+            ':keep_reports: false'
+          ])
+        end
+
+        it 'should configure ansible.cfg' do
```

Comment:
I guess this will fail since it needs a file to exist.

Reply:
That's configured here:
https://github.com/theforeman/puppet-foreman_proxy/blob/e1f8a20239bce5f458ef52d8001d5d6f3e29977d/manifests/plugin/ansible.pp#L59-L65

Why do you expect it to configure it as part of this plugin?

Reply:
Well, we don't want to fully configure this file, we just need to add two entries: `report_type` and `proxy_url` to actually enable redirection of the reports.